thumbor/error_handlers/sentry.py

```diff
@@ -35,7 +35,7 @@ def get_modules(self):
                 res_mod = pkg_resources.get_distribution(module)
                 if res_mod is not None:
                     resolved[module] = res_mod.version
-            except pkg_resources.DistributionNotFound:
+            except Exception:
```

Comment:
was hoping to use `except (pkg_resources.DistributionNotFound, pkg_resources.RequirementParseError)`, but looks like `RequirementParseError` does not exist in setuptools used on travis.